items.md

```diff
@@ -4,8 +4,9 @@
 ## Main Endpoint
 **/api/core/items**   
 
-Provide access to the items (DBMS based). It returns the list of existent items.
-This endpoint is reserved to the administrators the common practice is to navigate to the single items via a discovery search or browse
+Provide access to the items (DBMS based). This endpoint is reserved to the administrators the common practice is to navigate to the single items via a discovery [search](search-endpoint.md) or [browse](browses.md).
+
+Currently, this endpoint only returns "archived" items which are NOT withdrawn. However, withdrawn items and Workspace/Workflow items are findable via [search endpoints](search-endpoint.md). In the future, this endpoint is likely to be updated to include all items, regardless of status, see https://github.com/DSpace/DSpace/issues/3325
```

Comment:
Do you like to add
Template item are findable only via the corresponding collection

Reply:
Added that note about template items. Thanks!